src/constants/customer-form-config.js

```diff
@@ -39,7 +39,11 @@ export const hooks = {
   onSuccess(form: Object) {
     // get initial values of the form, to see if we are creating or updating modalStyle
     const initialsValue = form.initials();
-    if (initialsValue.name && initialsValue.url && initialsValue.industry) {
+    if (
+      initialsValue.name === '' &&
```

Comment:
if(initialsValue.name) ?

Reply:
mobx-react-form will automatically fill empty input with empty string, so `initialsValue.name === ''` is needed